lms/djangoapps/course_api/tests/test_views.py

```diff
@@ -85,8 +85,13 @@ def test_as_inactive_user(self):
         self.verify_response(params={'username': inactive_user.username})
 
     def test_missing_username(self):
+        response_to_anonymous = self.verify_response(expected_status_code=200)
```

Comment:
Is it? I'm retrieving a response without setting up the user, then getting the same response after I've set up the user.

Put another way, I believe this test would fail without my most recent changes. I agree that it is not comprehensive, but I don't think that it is a no-op unless I'm missing something (which is always a possibility).